combirepo/commandline_parser.py

```diff
@@ -312,7 +312,12 @@ def __set_program_run_parameters(self, arguments):
         else:
             logging.basicConfig(level=logging_level)
 
-        config_parser.initialize_config(arguments.config)
+        if len(arguments.triplets) == 0:
+            gen_init_config = True
+        else:
+            gen_init_config = False
+
+        config_parser.initialize_config(arguments.config, gen_init_config)
```

Comment:
For clarification: So we are generating a config if the user provides neither triplets through command line, nor configuration file, right?

Reply:
Yes, right.

Reply:
OK, thanks! Then I'm OK to merge this pull request as-is. Maybe we just need to improve commit message a bit (see above comment).

Reply:
Done